tests/Fixer/FunctionNotation/ReturnTypeDeclarationFixerTest.php

```diff
@@ -44,10 +44,6 @@ public function testFixWithDefaultConfiguration($expected, $input = null)
         $this->fixer->configure(null);
 
         $this->doTest($expected, $input);
-
-        $this->fixer->configure(array());
-
-        $this->doTest($expected, $input);
```

Comment:
Testing if `null` and `array` result in the same thing is testing if the `AbstractFixer` handles this correctly. I think this should be done in a diff. test or split up here into multiple test methods.